### PR TITLE
Ra 03 add courses

### PR DESCRIPTION
**Pull request by lincAlinc**

### CODE REVIEW

File: app/assets/stylesheets/scaffolds.scss [hunk 1]
```diff
@@ -0,0 +1,84 @@
+body {
```
Comment (deHelden) [inline app/assets/stylesheets/scaffolds.scss]: later this scss would cauze some problems with bootstrap.

File: app/views/courses/edit.html.slim [hunk 2]
```diff
@@ -0,0 +1,8 @@
+h1 Editing course
+
+== render 'form'
```
Comment (deHelden) [inline app/views/courses/edit.html.slim]: slim complete will render with one '=' symbol

File: app/views/courses/new.html.slim [hunk 3]
```diff
@@ -0,0 +1,5 @@
+h1 New course
+
+== render 'form'
```
Comment (deHelden) [inline app/views/courses/new.html.slim]: same here. One =

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -10,6 +10,8 @@
   </head>
 
   <body>
+  <div class="container">
```
Comment (deHelden) [inline app/views/layouts/application.html.erb]: . container

Comment (lincAlinc) [reply app/views/layouts/application.html.erb]: @deHelden, erb

File: spec/models/course_spec.rb [hunk 5]
```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe Course, type: :model do
```
Comment (deHelden) [inline spec/models/course_spec.rb]: WOW!
That is awesome :)

**Pull request merged**